UI_UX/Frontend/src/components/ExchangeRate.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+
+//value = -1 => loading state
+export const ExchangeRate = ({ value }) => {
+  let loadingIcon = undefined;
+  if (value == -1)

```

Comment:
When comparing values, always use `===`. In this case you are assuming the "invalid" value of the application is `-1`, but that might not be the case. Use shortcut conditionals instead, I'll show you how in the next comment.

Reply:
fixed